### PR TITLE
GOV: point CoC reports at CoC steering council subcomittee mailing list

### PR DESCRIPTION
**Pull request by tacaswell**
This is the last place we had `matplotlib@numfocus.org` in the docs. 

We hand funding from CZI EOSS4 for CoC training and I intend to stand up a proper CoC sub-committee, however for now the streering-council is a reasonable standin.

### CODE REVIEW

**Comment by tacaswell** [comment]
Needs https://github.com/matplotlib/governance/pull/26 to go in first so we have a working URL target.

**Review by timhoffm — APPROVED** [review]

**Comment by dopplershift** [comment]
It's merged. Waiting for this to leave draft to review.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -61,10 +61,13 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-matplotlib@numfocus.org (monitored by Thomas Caswell (\@tacaswell) and Hannah
-Aizenman (\@story645)) or a report can be made using the [NumFOCUS Code of Conduct report form][numfocus form].
-If community leaders cannot come to a resolution about enforcement, reports will be escalated to the NumFocus Code of Conduct committee (conduct@numfocus.org).
-All complaints will be reviewed and investigated promptly and fairly.
+[matplotlib-coc@numfocus.org](mailto:matplotlib-coc@numfocus.org)
+(monitored by the [CoC subcommittee](https://matplotlib.org/governance/people.html#coc-subcommitee)) or a
```
Comment (tacaswell) [inline CODE_OF_CONDUCT.md:65]: https://matplotlib.org/governance/people.html#coc-subcommittee is now live.

Comment (QuLogic) [inline CODE_OF_CONDUCT.md:65]:
```suggestion
(monitored by the [CoC subcommittee](https://matplotlib.org/governance/people.html#coc-subcommittee)) or a
```

**Review by dopplershift — APPROVED** [review]

**Pull request merged**